### ISSUE TITLE
Item has no value Error or just crashing

### ISSUE THREAD

**Issue by Tomshiii** [TheArkive/DLFile_ahk2#1]
I've recently noticed that attempting to use the lib will either cause the below error or will just crash the script that attempts to run it when the link is changed to a github release download link.
Not sure if maybe one of the recent ahk updates broke things or if it's just github doing github things but just thought I would bring it to your attention :)

test code used is the example in the lib itself but change the url to: `"https://github.com/AutoHotkey/AutoHotkey/releases/download/v2.0.5/AutoHotkey_2.0.5_setup.exe"`

lmk if I'm just a dummy and this is expected

![image](https://github.com/TheArkive/DLFile_ahk2/assets/53557479/df76cf2e-c62e-4285-9e67-87a48bd79703)

**Comment by TheArkive** [comment]
No, you're no dummy.  I've come across this too.  It's on my to-do list.

**Comment by Tomshiii** [comment]
> No, you're no dummy.  I've come across this too.  It's on my to-do list.

all good gamer, appreciate the update. Take your time (:
